docs/dashboards/overview-dashboard.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[overview-dashboard]]
+= Overview dashboard
+
+The Overview dashboard provides a high-level snapshot view of detections, external alerts, and event trends. These trends are useful to assess overall system health and find anomalies that may require further investigation.
```

Comment:
```suggestion
The Overview dashboard provides a high-level snapshot of detections, external alerts, and event trends. It can help you assess overall system health and find anomalies that may require further investigation.
```